scripts/updatekinto.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const DEBUG = true;
+
+const got = require("got");
+
+const SERVER = "https://settings-writer.prod.mozaws.net/v1";
+const CID = "fxmonitor-breaches";
+/* eslint-disable-next-line no-process-env */
+const USERNAME = process.env.KINTO_USERNAME;
+/* eslint-disable-next-line no-process-env */
+const PASSWORD = process.env.KINTO_PASSWORD;
+if (!USERNAME || !PASSWORD) {
+  console.error("Please set credentials in the environment.");
+  return;
+}
+
+const AUTH = Buffer.from(`${USERNAME}:${PASSWORD}`).toString("base64");
+
+async function run() {
+  const RemoteSettingsBreachesSet = new Set((await got(
+    "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/fxmonitor-breaches/records",
+    { json: true }
+  )).body.data.map(b => b.Name));
+
+  const hibp_breaches = (await got(
+    "https://haveibeenpwned.com/api/v2/breaches",
+    { json: true }
+  )).body;
+
+  const new_breaches = [];
+
+  for (const breach of hibp_breaches) {
+    if (breach.IsSpamList || breach.IsRetired || !breach.IsVerified || !breach.Domain) {
+      continue;
+    }
+
+    if (RemoteSettingsBreachesSet.has(breach.Name)) {
+      continue;
+    }
+
+    new_breaches.push(breach);
+  }
+  for (const breach of new_breaches) {
+    const data = {
+      Name: breach.Name,
+      Domain: breach.Domain,
+      BreachDate: breach.BreachDate,
+      PwnCount: breach.PwnCount,
+      AddedDate: breach.AddedDate,
```

Comment:
Do we need others like `.LogoPath` and `.DataClasses` and `.Title`?

https://github.com/mozilla/blurts-server/blob/a7a5730bf0bd83d6183c185f21fde70e98b99415/hibp.js#L88-L89

Reply:
Nope, our Kinto collection only maintains these 5 properties. We can add new properties if/when the need arises - the data is sync'd with Firefox clients and I'd like to avoid having users download unnecessary data.